.github/workflows/legacy.yml

```diff
@@ -0,0 +1,77 @@
+name: Legacy
+
+on:
+  push:
+    branches: [main]
+    paths:
+      - .github/workflows/**
+      - api/**
```

Comment:
Why run on api changes? Currently there are no tests that would verify things breaking in legacy if the API changed

Reply:
I expect both to be tightly coupled together during the transition to django. We already add more coupling with #1644 or #1787 this was more a way to make sure we don't break legacy when changing something on the api side. 

I can remove this path though.

Reply:
Not necessary with #1644. With that, we will need to test both